lib/v2/gov/radar.js

```diff
@@ -688,6 +688,17 @@ module.exports = {
             },
         ],
     },
+    'stats.gov.cn': {
+        _name: '国家统计局',
+        std: [
+            {
+                title: '国家统计局',
+                docs: 'https://docs.rsshub.app/government.html#guo-jia-tong-ji-ju',
+                source: ['/'],
+                target: (params, url) => `/gov/stats/${new URL(url).match(/stats\.gov\.cn\/(.*)/)[1]}`,
```

Comment:
`match` does not work on `object`
```suggestion
                target: (params, url) => `/gov/stats/${new URL(url).href.match(/stats\.gov\.cn\/(.*)/)[1]}`,
```